benchmark/sstable_read_test.go

```diff
@@ -52,6 +52,7 @@ func BenchmarkSSTableReadIndexTypes(b *testing.B) {
 		name   string
 		loader sstables.IndexLoader
 	}{
+		{"map", &sstables.SortedMapIndexLoader{ReadBufferSize: 4096}},
```

Comment:
really curious about the performance when you have the GET benchmark ready!

Reply:
![image](https://github.com/user-attachments/assets/f2fad507-5413-4048-b596-5c5b19a75736)

the get time unsurpriselly better with the map version. 1605 get_time_ns/record

Reply:
that's awesome!